ClientLib/src/main/java/com/o3dr/services/android/lib/drone/companion/solo/tlv/SoloPanoStatus.java

```diff
@@ -0,0 +1,70 @@
+package com.o3dr.services.android.lib.drone.companion.solo.tlv;
+
+import android.os.Parcel;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Created by phu on 7/17/16.
+ */
+public class SoloPanoStatus extends TLVPacket {
+
+    private int currentStep;
```

Comment:
@phmagic no need to cast a `byte` to an `int` since it's not a downcast, so it's fine to make this variable a `byte`.